Lib/test/test_argparse.py

```diff
@@ -2093,6 +2093,26 @@ class TestActionExtend(ParserTestCase):
     ]
 
 
+class TestNegativeNumberAction(ParserTestCase):
```

Comment:
```suggestion
class TestNegativeNumber(ParserTestCase):
```
nit: since it doesn't apply to the `action` keyword